flake.nix

```diff
@@ -284,6 +272,8 @@
               PLONK_WASM_WEB
               KIMCHI_STUBS
               KIMCHI_STUBS_STATIC_LIB
+              MARLIN_PLONK_STUBS
```

Comment:
It should not be re-introduced. The variables have been renamed into kimchi_stubs

Reply:
In this pr the fact that they were missing was breaking ci:
https://github.com/o1-labs/o1js/pull/2131/commits/d3af820c1b4e77316ed6b3227373301f7bf07128

I guess updating mina would fix this if it didn't break other stuff.